rules/windows/credential_access_remote_sam_secretsdump.toml

```
@@ -49,9 +50,16 @@ be monitored by the security team.
 
 - Initiate the incident response process based on the outcome of the triage.
 - Isolate the involved hosts to prevent further post-compromise behavior.
-- Scope compromised credentials and disable the accounts.
-- Reset the passwords of compromised accounts.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure that all compromised accounts
+compromised by the attacker are known. Reset the password for these accounts and other potentially compromised credentials,
+like email, business systems, and web services.
 - Determine if other hosts were compromised.
+- Run a full scan using the antimalware tool in place. This scan can reveal additional artifacts left in the system,
+persistence mechanisms, and malware components.
+- Reimage the host operating system or restore the compromised files to clean versions.
+- Ensure that the machine has the latest security updates and is not running legacy Windows versions.
```

"Legacy" is a little vague -- "unsupported" or maybe "deprecated"?
```suggestion
- Ensure that the machine has the latest security updates and is not running unsupported Windows versions.
```